mlflow/R/mlflow/R/tracking-rest.R

```diff
@@ -10,6 +10,18 @@ mlflow_rest_timeout <- function() {
   timeout(getOption("mlflow.rest.timeout", 60))
 }
 
+
+tryParseResponseAsText <- function(response) {
```

Comment:
I think we've mostly used `snake_case` elsewhere, can we rename?

Reply:
Makes sense, will do.